packages/management/tkr-service/bundle/config/upstream/certificates.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  labels:
+  name: tkr-resolver-cluster-webhook-selfsigned-issuer
+  namespace: tanzu-system
```

Comment:
Should this be in the newly created `tkr-system` namespace?

Reply:
The namespace would be replaced using ytt overlay with namespace value from `values.yaml`.

Reply:
I will change the namespace to `tkr-system` to avoid confusion as we are planning to have `tkr-system` as default namespace

Reply:
Updated